src/main/java/com/nha/abdm/wrapper/hip/hrp/GatewayCallbackController.java

```diff
@@ -260,4 +265,29 @@ private ResponseEntity<GatewayCallbackResponse> handleIllegalDataStateException(
             .build(),
         HttpStatus.INTERNAL_SERVER_ERROR);
   }
+
+  /**
+   * Callback from ABDM gateway of consent to dataTransfer
+   * @param hipConsentNotification Has careContexts details for dataTransfer
+   */
+  @PostMapping({"/v0.5/consents/hip/notify"})
+  public void initiateDataOnNotify(@RequestBody HIPConsentNotification hipConsentNotification) {
+    if (hipConsentNotification != null) {
+      workflowManager.initiateDataOnNotify(hipConsentNotification);
+    } else log.debug("Error in response of Data Notify");
+  }
+
+  /**
+   * Callback from ABDM gateway for dataTransfer to HIU
+   * @param hipHealthInformationRequest Has keys for encryption and dataPushURL of HIU
+   */
+  @PostMapping({"/v0.5/health-information/hip/request"})
+  public void DataRequestResponse(
+      @RequestBody HIPHealthInformationRequest hipHealthInformationRequest) {
+    if (hipHealthInformationRequest != null) {
```

Comment:
It can be modified in the next PR, the constants that are mentioned are used by ABDM, but we should give the wrapper flexibility, so by next PR, changes can be made.